src/register/register.ts

```diff
@@ -30,9 +31,16 @@ export class Register {
    */
   private static registers: { [key: string]: IRegisterContent } = {
     '"': { text: "", registerMode: RegisterMode.CharacterWise },
-    '*': { text: "", registerMode: RegisterMode.CharacterWise }
+    '*': { text: "", registerMode: RegisterMode.CharacterWise },
+    '+': { text: "", registerMode: RegisterMode.CharacterWise }
   };
 
+  private static systemClipboardRegisters: string[] = ["*", "+"];
+
+  public static isSystemClipboardRegister(register: string): boolean {
```

Comment:
Good call! I've switched the check to look up the `isClipboardRegister` property on the register.

(For consistency's sake, I also renamed the check from `Register.isSystemClipboardRegister` to just `Register.isClipboardRegister`. Saves a few keystrokes too!)